src/plugins/vis_type_timeline/server/series_functions/graphite_helper.js

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
this makes this accessible within the visualization which should be fine.

Reply:
move helper to helpers folder to solve. nice catch.